mars/deploy/local/session.py

```diff
@@ -148,26 +150,34 @@ def run(self, *tileables, **kw):
         else:
             return self.fetch(*tileables)
 
-    def fetch(self, *tileables):
+    def fetch(self, *tileables, **kw):
```

Comment:
We must keep consistency about `fetch` for all kinds of sessions. If `kw` is necessary, make sure all `fetch` have the same signature.

Reply:
Will fix.